skyvern/forge/sdk/db/client.py

```diff
@@ -2250,3 +2252,96 @@ async def get_workflow_run_blocks(
                 convert_to_workflow_run_block(workflow_run_block, task=tasks_dict.get(workflow_run_block.task_id))
                 for workflow_run_block in workflow_run_blocks
             ]
+
+    async def get_active_persistent_browser_sessions(self, organization_id: str) -> List[PersistentBrowserSession]:
+        """Get all active persistent browser sessions for an organization."""
+        async with self.Session() as session:
+            result = await session.execute(
+                select(PersistentBrowserSessionModel).where(
+                    PersistentBrowserSessionModel.organization_id == organization_id,
```

Comment:
the convention in our code base (see other functions in this file) is using `select(Model).filter_by` or `select(Model).filter`